### PR TITLE
Test last convert method

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: test/core.jl [hunk 1]
```diff
@@ -135,4 +135,6 @@
     @test Tuple{2, 3, 5} != StaticArrays.Size{(2, 3, 4)}
     @test StaticArrays.Size{(2, 3, 4)} != Tuple{2, 3, 5}
     @test StaticArrays.check_length(2) == nothing
+
+    @test convert(Tuple, @SVector [1]) == (1,)
```
Comment (mschauer) [inline test/core.jl]: Better, ``@test convert(Tuple, @SVector [2]) === (2,)``
because e.g.
```
julia> Size(@SVector [1]) == (1,)
true
```

Comment (kshyatt) [inline test/core.jl]: Great idea, updated!

File: test/core.jl [hunk 2]
```diff
@@ -135,4 +135,6 @@
     @test Tuple{2, 3, 5} != StaticArrays.Size{(2, 3, 4)}
     @test StaticArrays.Size{(2, 3, 4)} != Tuple{2, 3, 5}
     @test StaticArrays.check_length(2) == nothing
+
+    @test convert(Tuple, @SVector [2]) == (2,)
```
Comment (andyferris) [inline test/core.jl]: I tend to use `===` for isbits immutables.

Comment (mschauer) [inline test/core.jl]: Cue my comment ;-)

**Pull request merged**